src/main/java/net/countercraft/movecraft/combat/features/tracking/DamageTracking.java

```diff
@@ -34,6 +33,14 @@ public static void load(@NotNull FileConfiguration config) {
         DamageTimeout = config.getInt("DamageTimeout", 300);
     }
 
+    //This is where it stores which players to obfuscate kill credits from.
+    public static void setKillCredMap(Player p, Boolean b) {
+        hideCredMap.put(p,b);
+    }
+    public static Map<Player, Boolean> getKillCredMap() {
+        return hideCredMap;
+    }
```

Comment:
Instead of returning the map itself, let's make a function `public Boolean getKillCred(Player)` to access the setting of a player.  If there is no entry, return false without inserting it into the map.

Reply:
Fixed